src/util/XDRCereal.cpp

```diff
@@ -37,17 +37,19 @@ cereal_override(cereal::JSONOutputArchive& ar,
 }
 
 void
-cereal_override(cereal::JSONOutputArchive& ar, const stellar::Asset& asset,
-                const char* field)
+cereal_override(cereal::JSONOutputArchive& ar, stellar::Asset const& asset,
+                char const* field)
+{
+    xdr::archive(ar, assetToCerealNVP(asset), field);
+}
+
+NVPTuple
+assetToCerealNVP(stellar::Asset const& asset)
 {
-    xdr::archive(
-        ar,
-        std::make_tuple(
-            cereal::make_nvp("assetCode", stellar::assetToString(asset)),
-            cereal::make_nvp(
-                "issuer",
-                asset.type() == stellar::ASSET_TYPE_NATIVE
-                    ? "SDF"
-                    : stellar::KeyUtils::toStrKey(stellar::getIssuer(asset)))),
-        field);
+    return std::make_tuple(
```

Comment:
Same thing as mentioned in the previous commit about rendering native asset.

Reply:
Fixed!